docs/pages/radiogroup.js

```diff
@@ -0,0 +1,986 @@
+        name={generatedDocGen.RadioGroup.displayName}
+        description={generatedDocGen.RadioGroup?.description}
+        defaultCode={`
+      function RadioButtonExample() {
+        const [favorite, setFavorite] = React.useState();
+
+        return (
+          <RadioGroup legend="Gender" id="header-example">
+            <RadioGroup.RadioButton
+              checked={favorite === 'Female'}
+              id="genderFemale"
+              label="Female"
+              name="gender-pref"
+              onChange={() => setFavorite('Female')}
+              value="Female"
+            />
+            <RadioGroup.RadioButton
+              checked={favorite === 'Male'}
+              id="genderMale"
+              label="Male"
+              name="gender-pref"
+              onChange={() => setFavorite('Male')}
+              value="Male"
+            />
+            <RadioGroup.RadioButton
+              checked={favorite === 'Non-binary'}
+              id="genderNon-binary"
+              label="Non-binary"
+              name="gender-pref"
+              onChange={() => setFavorite('Non-binary')}
+              value="Non-binary"
+            />
+            <RadioGroup.RadioButton
+              checked={favorite === 'Prefer not to state'}
+              id="genderPrefer not to state"
+              label="Prefer not to state"
+              name="gender-pref"
+              onChange={() => setFavorite('Prefer not to state')}
+              value="Prefer not to state"
+            />
+          </RadioGroup>
+        );
+      }
+      `}
+      />
+
+      <GeneratedPropTable generatedDocGen={generatedDocGen.RadioGroup} />
+      <MainSection name="Subcomponents">
+        <MainSection.Subsection
+          title={generatedDocGen?.RadioGroupButton.displayName}
+          description={generatedDocGen?.RadioGroupButton.description}
+        >
+          <GeneratedPropTable
+            Component={RadioGroup?.RadioButton}
+            name="RadioGroup.RadioButton"
+            id="RadioGroup.RadioButton"
+            generatedDocGen={generatedDocGen.RadioGroupButton}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <MainSection name="Usage guidelines">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            title="When to use"
+            description={`
+          - In a list, form or table, to present users with multiple, related options where only one option can be selected.
+          - When selection doesn’t take immediate effect and requires form submission.
+        `}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            title="When not to use"
+            description={`
+          - Situations where users can select multiple options. Use [Checkbox](/checkbox) instead.
+          - When there is only one item to select or deselect. Use Checkbox instead.
```

Comment:
Checkbox with link, doesnt have one here, unless your avoiding the second link on purpose